### PR TITLE
Use a WHATWG URL spec compliant URL parser - see https://url.spec.wha…

### PR DESCRIPTION
**Pull request by DeepDiver1975**
…twg.org/ - this is necessary to parse the redirect url as close as browsers do.

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
Damn - https://packagist.org/packages/rowbot/url requires php-gmp which is not in the ci containers and also not part of the usual owncloud installation ...

**Comment by DeepDiver1975** [comment]
> requires php-gmp which is not in the ci containers

fixed by https://github.com/owncloud-ci/php/pull/108



> not part of the usual owncloud installation

wrong - THX @pako81 - php-gmp is a documented requirement since oc 9.0

File: .drone.starlark [hunk 1]
```diff
@@ -9,6 +9,8 @@ config = {
 		'master'
 	],
 
+    'appInstallCommand': 'make vendor',
```
Comment (PVince81) [inline .drone.starlark]: indent

**Comment by DeepDiver1975** [comment]
Overriding CLA and Codecov -> merge

**Pull request merged**